plugins/node/opentelemetry-instrumentation-ioredis/test/ioredis.test.ts

```diff
@@ -563,7 +561,8 @@ describe('ioredis', () => {
 
         const attributes = {
           ...DEFAULT_ATTRIBUTES,
-          [SemanticAttributes.DB_STATEMENT]: `evalsha bfbf458525d6a0b19200bfd6db3af481156b367b 1 ${testKeyName}`,
+          [SemanticAttributes.DB_STATEMENT]:
+            'evalsha bfbf458525d6a0b19200bfd6db3af481156b367b',
```

Comment:
also here - the new `db.statement` value is not correct. Might be worth mentioning in a comment to communicate that this value is not a bug and ignored intentionally for now.